src/test/java/Homework16.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public static void Registration() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+        // find element-registration
+        // click on registration
+        WebElement registrationLink = driver.findElement(By.cssSelector("[id='hel']"));
+        registrationLink.click();
+        String registrationURL = "https://bbb.testpro.io/registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(), registrationURL);
+
+        driver.quit();
+    }
+}
```

Comment:
You got this problem  `class LoginTests`  , please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210048483-c8e456aa-416f-431e-99fd-5c750b8c662d.png)